shardingsphere-agent/shardingsphere-agent-plugins/shardingsphere-agent-plugin-metrics/shardingsphere-agent-metrics-prometheus/src/test/java/org/apache/shardingsphere/agent/metrics/prometheus/collector/ProxyInfoCollectorTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.agent.metrics.prometheus.collector;
+
+import io.prometheus.client.Collector;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertThat;
+
+public final class ProxyInfoCollectorTest {
+    
+    @Test
+    public void assertCollect() {
+        ProxyInfoCollector proxyInfoCollector = new ProxyInfoCollector();
+        List<Collector.MetricFamilySamples> metricFamilySamples = proxyInfoCollector.collect();
+        assertThat(metricFamilySamples.size(), org.hamcrest.Matchers.greaterThan(0));
```

Comment:
Use static import for Matchers.

Reply:
remove it will not build locally.

Reply:
I think this is not elegant. Consider using `isEmpty()`.

Reply:
good suggestion.